setup.py

```diff
@@ -58,7 +58,7 @@ def get_version():
     ],
     install_requires=requirements,
     extras_require={
-        "docs": ["sphinx", "sphinx_rtd_theme", "recommonmark", "sphinxcontrib-bibtex", "sphinx_markdown_tables", "nbsphinx", "IPython"],
+        "docs": ["sphinx", "sphinx_book_theme", "recommonmark", "sphinxcontrib-bibtex", "sphinx_markdown_tables", "nbsphinx", "IPython"],
```

Comment:
We are still using rtd_theme, aren't we?

Reply:
Well spotted, thanks.